prow/cmd/branchprotector/protect.go

```diff
@@ -173,6 +174,7 @@ func (p *Protector) ConfigureBranches() {
 // Protect branches specified in the presubmit and branch-protection config sections.
 func (p *Protector) Protect() {
 	bp := p.cfg.BranchProtection
+	log.Println(bp)
```

Comment:
I think this will default to the equivalent of using the `%v` format specifier which doesn't show field names.
Using `log.Printf("%#v", bp)` or marshaling the struct into yaml or json will be easier to read when this config becomes complicated.

Reply:
Done